app/tests/web/db/test_crud.py

```diff
@@ -140,43 +476,107 @@ def test_search_archives_by_email(test_data, db_session):
 def test_max_query_limit(test_data, db_session):
     from app.web.config import ALLOW_ANY_EMAIL
 
-    assert len(crud.search_archives_by_url(db_session, "https://example", ALLOW_ANY_EMAIL, [], False)) == 25
-    assert len(crud.search_archives_by_url(db_session, "https://example", ALLOW_ANY_EMAIL, True, True, limit=1000)) == 25
+    assert (
+        len(
+            crud.search_archives_by_url(
+                db_session, "https://example", ALLOW_ANY_EMAIL, [], False
+            )
+        )
+        == 25
+    )
+    assert (
+        len(
+            crud.search_archives_by_url(
+                db_session,
+                "https://example",
+                ALLOW_ANY_EMAIL,
+                True,
+                True,
+                limit=1000,
+            )
+        )
+        == 25
+    )
 
-    assert len(crud.search_archives_by_email(db_session, "rick@example.com")) == 25
-    assert len(crud.search_archives_by_email(db_session, "rick@example.com", limit=1000)) == 25
+    assert (
+        len(crud.search_archives_by_email(db_session, "rick@example.com")) == 25
+    )
+    assert (
+        len(
+            crud.search_archives_by_email(
+                db_session, "rick@example.com", limit=1000
+            )
+        )
+        == 25
+    )
 
 
 def test_soft_delete(test_data, db_session):
     # none deleted yet
-    db_session.query(models.Archive).filter(models.Archive.id == "archive-id-456-0").first() is not None
-    assert db_session.query(models.Archive).filter(models.Archive.deleted == True).count() == 0
+    assert (
```

Comment:
Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.